src/transformer/mockDefiner/uniqueIdentifier.ts

```diff
@@ -0,0 +1,5 @@
+import * as ts from 'typescript';
+
+export function PrivateIdentifier(text: string): ts.Identifier {
+  return ts.createIdentifier(`ɵ${text}`);
```

Comment:
don't we have other places where we use that character? could we reuse this function?

Reply:
done :)